test/e2e/scheduling/priorities.go

```diff
@@ -316,6 +378,21 @@ func createBalancedPodForNodes(f *framework.Framework, cs clientset.Interface, n
 	return nil
 }
 
+// createImagePodForNode creates a pod on the given node with given images and waits until the pod's running.
+func createImagePodForNode(f *framework.Framework, cs clientset.Interface, ns string, node v1.Node, images []imageutils.ImageConfig) error {
+	err := testutils.StartPods(cs, 1, ns, string(uuid.NewUUID()),
+		*initImagePod(f, imagePodConfig{
+			Name:   "",
```

Comment:
why pass an empty name here?

Reply:
btw, others lgtm.

Reply:
Addressed - I believed it was just an "image-puller" pod with just the uuid name would work - now added "image-pod" name. Thanks!